presto-main/src/main/java/com/facebook/presto/sql/gen/ExpressionCompiler.java

```diff
@@ -119,13 +120,13 @@ public CacheStatsMBean getCursorProcessorCache()
 
     public Supplier<PageProcessor> compilePageProcessor(SqlFunctionProperties sqlFunctionProperties, Optional<RowExpression> filter, List<? extends RowExpression> projections)
     {
-        return compilePageProcessor(sqlFunctionProperties, filter, projections, Optional.empty());
+        return compilePageProcessor(sqlFunctionProperties, filter, projections, false, Optional.empty());
```

Comment:
If you are hard coding false here then this should be annotated `@VisibleForTesting`

Reply:
done